apm-collector/apm-collector-boot/src/main/resources/application.yml

```diff
@@ -68,7 +68,11 @@ storage:
     indexShardsNumber: 2
     indexReplicasNumber: 0
     highPerformanceMode: true
-    ttl: 7
+    # Set a timeout on metric data. After the timeout has expired, the metric data will automatically be deleted.
+    minuteMetricDataTTL: 45 # Unit is minute
+    hourMetricDataTTL: 36 # Unit is hour
+    dayMetricDataTTL: 45 # Unit is day
+    monthMetricDataTTL: 18 # Unit is month
```

Comment:
I suggest you remove `Data` from the key. Because metric is data related only.

Reply:
From my code review, remove trace based on `minuteMetricDataTTL`. Then I recommend you to provide a traceTTL.

Reply:
> I suggest you remove Data from the key. Because metric is data related only.

I'm not sure who is correct? I search `metric data` in Baidu and Google, The `metric data` and `metric` in articles are on the same frequency.

Reply:
Because almost all settings in storage is data related.  Your decision.